### DISCUSSION TITLE
Need help with DFP-34X-2C2 not showing VLAN with omcicli mib get 84

### DISCUSSION THREAD

**Discussion by techtito** [Anime4000/RTL960x#270] [General]
Hello guys, can someone please help me?
My ISP is "Oi/V.Tal" here in Brazil and uses in my region only Huawei HGU, i also checked my OLT using ODI Stick and report its an Huawei too.
I have an friend trying to help me in this, last day we run Collect Fault on Huawei HGU and seems its reported the OLT uses for autentication:
Type:Authorization,48575443C661499D-HG8245Q2-00259E

SN - ONU MODEL - OUI

we tried everything also:
Mac Clone + Correct new Mac Key

But never returns VLAN on ODI.

We started to think the OLD need the SN must be placed in this format:
48575443C661499D
not this:
HWTCC661499D

Someone can say if is possible to set SN on this format?
48575443C661499D

Or have any ideas what can be happening?

**Comment by Anime4000** [comment]
in OMCI stack on ODI, SN always 4 ASCII + 4 HEX
when OMCI exchange between ODI and OLT, it is in Bytecode

have you tried change RTK value?

**Reply by techtito** [reply]
Thanks for reply, i will try using RTK values, can you please tell me if the commands is:

flash set RTK_DEVID_MANUFACTURER 
flash set RTK_DEVID_OUI
flash set RTK_DEVID_PRODUCTCLASS
flash set RTK_DEVINFO_SPECVER
flash set RTK_DEVINFO_SWVER
flash set RTK_DEVINFO_HWVER

Or something like this?

And still about my region here in Brazil, my ISP uses an Huawei OLT, and i have a new discovery: https://www.tripleoxygen.net/wiki/ont/vsol/v2802rh

This guy is Triple Oxygen and he had to make a modification on the VSOL firmware for enable MIBs/MEs proprietary from OLTs of Huawei:

Here is changelog for 1.9.4v - 
1.9.4
Habilita MIBs/MEs proprietários da Huawei. Útil para regiões com OLTs Huawei que exigem estas extensões. Certifique-se de emular um aparelho Huawei neste caso. Não impacta caso a OLT que atenda seu link não seja desta marca.

So i think the major problem is that, i have buy one VSOL to test and see if it works, if work ok, did you can research something about this  MIBs/MEs thing for ODI also?

**Comment by techtito** [comment]
So i going to test RTK today later, can you please help me on values string?
On Collect Fault function in HGU its reports:
Type:Authorization,48575443C661499D-HG8245Q2-

OUI is: 00259E - NOT 6 digits of mac address a lot of people say it over internet.

![Sem título](https://github.com/Anime4000/RTL960x/assets/166328297/73f59506-a33e-47e0-a912-9eee606552bf)
![Sem título](https://github.com/Anime4000/RTL960x/assets/166328297/1cf039b1-f401-48e8-a91f-2acd388e9046)
![Sem título](https://github.com/Anime4000/RTL960x/assets/166328297/3aa62aa9-c98d-4af1-9073-67b03a443453)

Collect Fault also:

[collectFaultFinish.txt](https://github.com/Anime4000/RTL960x/files/14909806/collectFaultFinish.txt)


Thank you very much.

**Reply by techtito** [reply]
aditional info, this guy "FelipeSS" have sucess in the same situation of me, but using VSOL:
https://forum.adrenaline.com.br/threads/eliminar-ou-trocar-onu-ont-hgu-da-operadora-provedor.699243/page-79#post-1076776505

ONT Original: HG8145V5-V2
OLT: Huawei
ONU Utilizada: VSOL V2802RH

Configurações realizadas:
Firmware modificado [Triple Oxygen](https://www.tripleoxygen.net/wiki/ont/vsol/v2802rh) para rodar em modo SFU o Vsol (em breve ele coloca no site)

flash set GPON_SN HWTCxxxxxxxx
flash set PON_VENDOR_ID HWTC
cfgmib set OMCI_OLT_MODE 1
cfgmib set OMCC_VER 0x86
cfgmib set OMCI_TM_OPT 2
cfgmib save
restart

**Comment by canope-25** [comment]
> So i going to test RTK today later, can you please help me on values string?
> 
> On Collect Fault function in HGU its reports:
> 
> Type:Authorization,48575443C661499D-HG8245Q2-
> 
> 
> 
> OUI is: 00259E - NOT 6 digits of mac address a lot of people say it over internet.
> 
> 
> 
> ![Sem título](https://github.com/Anime4000/RTL960x/assets/166328297/73f59506-a33e-47e0-a912-9eee606552bf)
> 
> ![Sem título](https://github.com/Anime4000/RTL960x/assets/166328297/1cf039b1-f401-48e8-a91f-2acd388e9046)
> 
> ![Sem título](https://github.com/Anime4000/RTL960x/assets/166328297/3aa62aa9-c98d-4af1-9073-67b03a443453)
> 
> 
> 
> Collect Fault also:
> 
> 
> 
> [collectFaultFinish.txt](https://github.com/Anime4000/RTL960x/files/14909806/collectFaultFinish.txt)
> 
> 
> 
> 
> 
> Thank you very much.
> 
> 
> 
> 
> 
> 

Hi , seems to be basic ascii conversion to HEX: 
 48 => H , 57 => W , 54 => T 

![image](https://github.com/Anime4000/RTL960x/assets/64278388/45d7c6fa-488e-49d6-9ddd-e8bba2d11141)

**Reply by TakasiSHIH** [reply]
I also have the same problem.....
My system is ALCATEL-7360FX-8 and H660WM.
But I can not get the VLAN with omcicli mib get 84.

What can I do for this case.....QQ

![unnamed](https://github.com/Anime4000/RTL960x/assets/3366038/2ac26113-175f-42d3-a4f3-50eed095e9ff)

**Comment by kelt84** [comment]
Hello everyone, tell me, maybe someone knows how to solve this issue, I bought a DFP-34X-2C2 and reflashed it with all available firmware, the situation is as follows, authentication with the provider is carried out by the serial number of the device, when entering the serial number and then saving, it asks to enter the PLOAM password, but since authentication is carried out by the serial number, the PLOAM field must be left empty, how can this be done?
